readme.md

```diff
@@ -727,7 +811,7 @@ An object that has a `concat` function that combines it with another object of t
 
 ## Foldable
 
-An object that has a `reduce` function that can transform that object into some other type.
+An object that has a `reduce` function that applies a function against an accumulator and each element in the array (from left to right) to reduce it to a single value.
```

Comment:
Perhaps the detail about left to right can be omitted as there is also right to left reduce?

In maths, the right to left is actually more common as it is how the functions compose.